src/modules/vtol_att_control/vtol_att_control_params.c

```diff
@@ -213,12 +213,24 @@ PARAM_DEFINE_FLOAT(VT_F_TRANS_DUR, 3.0f);
  *
  * @unit s
  * @min 0.00
- * @max 10.00
+ * @max 20.00
  * @increment 1
  * @decimal 2
  * @group VTOL Attitude Control
  */
-PARAM_DEFINE_FLOAT(VT_B_TRANS_DUR, 2.0f);
+PARAM_DEFINE_FLOAT(VT_B_TRANS_DUR, 4.0f);
+
+/**
+ * Back transition MC motor ramp up time
+ *
+ * @unit s
+ * @min 0.00
+ * @max 20.00
+ * @increment 1
+ * @decimal 2
+ * @group VTOL Attitude Control
+ */
+PARAM_DEFINE_FLOAT(VT_B_TRANS_RAMP, 2.0f);
```

Comment:
I think we do, all time based loops are parameterized. I agree we are drowning in params but i think that calls for a different approach to params. but something like this will change for smaller and larger vtols